services/director-v2/tests/unit/with_swarm/test_dynamic_sidecar_scheduler_docker_api.py

```diff
@@ -0,0 +1,93 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+
```

Comment:
The name of this test does no t match the path of the python module it tests

``services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_api.py`` -->
``test_modules_dynamic_sidecar_docker_api.py``